### ISSUE TITLE
Dynamis Windurst Statue Aggro Range

### ISSUE THREAD

**Issue by jrwhite811** [EdenServer/community#4123]
### Checklist

- [x ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x ] I have provided my client version in the details. (type /ver into your game window)

### Details

It should be possible to avoid aggroing the statues outside of Koru-Moru's Manor in Dynamis-Windurst (aka the Death House) by hugging the right wall, allowing you to skip this pull altogether. In our Dynamis run Sunday night it was not possible to do so anymore - the house aggros even while hugging the wall, which resulted in an almost full wipe for us. 

This strategy dates back to era, and is outlined on the dynamis-bums website here: http://www.dynamisbums.com/strategy/win.html and also in the first page of this forum post from 2004 http://www.dynamisbums.com/forum/showthread.php?15-Dynamis-Windurst.

This strat also used to work on Eden, as seen here starting at 1:42:40 https://www.youtube.com/watch?v=jelzH2k05sE

version 30181205_0

**Comment by Shoruto** [comment]
Additional information provided by Kittycat

Video - https://www.youtube.com/watch?v=j0En3a5KKIc

Evidence at 7:10 on video.

**Comment by perkuprightmeow** [comment]
More from Cat: 

In looking through old videos, I believe that the statue that will aggro now if you walk is in the wrong place by a few yalms. 
At 4:58 in this potato quality video, you can see that this statue is placed to the right of the banner on the house and stares toward the right corner of the walkway.
https://youtu.be/TCvhAo_QHRI?t=298
![image](https://user-images.githubusercontent.com/52988064/112188090-b28d5f80-8bd0-11eb-81e0-2b8a1e405d23.png)

The best representation I can find of its placement on Eden is at 1:42:38 in the following video, which shows the statue more ore less aligned on the right half of the banner, rather than to the right of the banner, and stares at the left corner of the walkway.
https://youtu.be/jelzH2k05sE?t=6152 
![image](https://user-images.githubusercontent.com/52988064/112188167-c33dd580-8bd0-11eb-8ef1-5fb011f194e9.png)

Testing these two different positions on my characters in Windurst Walls, an object on the bottom step to the right of the banner is at closest 15.1 yalms away from the left corner of the walkway, just over aggro range, allowing you to sneak by, which was possible in era. 
![image](https://user-images.githubusercontent.com/52988064/112188257-d6e93c00-8bd0-11eb-8453-921eedc8b5e3.png)

In its current spot, the object is about 14.7 yalms, which will cause result in aggro whenever trying to walk by. 
![image](https://user-images.githubusercontent.com/52988064/112188305-e23c6780-8bd0-11eb-9350-f34545e32981.png)
The distance plays a factor, but correct placement may also make the left corner of the walkway out of the statue’s field of view, but that would need more testing.
Tldr – this statue is in slightly the wrong spot, leading to aggro if you try to walk by it as you could in era.

**Comment by 59blargedy** [comment]
resolved with the April 5, 2021 update
